tests/cross/src/test/scala/tests/tokens/SemanticTokensSuite.scala

```diff
@@ -166,33 +230,20 @@ class SemanticTokensSuite extends BasePCSuite {
         |    <<override>>/*modifier*/ <<def>>/*keyword*/ <<method>>/*method*/(<<adf>>/*parameter*/: <<String>>/*type*/): <<Int>>/*class,abstract*/ = <<321>>/*number*/
         |  }
         |}""".stripMargin,
+    compat = Map(
```

Comment:
I've added comments, it's because `+` (and other operators on numbers) and traits are not abstract, but they were in scala 2. Actually , maybe we want `traits` to be abstract (small change)?

Reply:
It looks like  `+` is abstract still, it's the same 2.13 Scala library, no?

Reply:
Look at `number literal, Static` test, `+` there is not abstract. What do you think about `traits`? Should we make them abstract to look the same like in Scala 2?